### PR TITLE
workflows: do not trigger a build for a merge PR

### PR DESCRIPTION
**Pull request by iMichka**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/linuxbrew-core/blob/master/CONTRIBUTING.md)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/linuxbrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?
- [ ] Have you included the output of `brew gist-logs <formula>` of the build failure if your PR fixes a build failure. Please quote the exact error message.

-----

### CODE REVIEW

**Comment by dawidd6** [comment]
> For a pull_request event, only branches and tags on the base are evaluated.

https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#onpushpull_requestbranchestags

**Comment by iMichka** [comment]
That's ... weird. Ok, too bad.

Do you think there is another way to prevent that build to happen?

**Comment by dawidd6** [comment]
Maybe something like:.

```yaml
if: startsWith(github.event.pull_request.title, 'Merge')
```

File: .github/workflows/build-bottles.yml [hunk 1]
```diff
@@ -7,6 +7,7 @@ on:
 
 jobs:
   build-bottles:
+    if: startsWith(github.event.pull_request.title, 'merge') == false
```
Comment (dawidd6) [inline .github/workflows/build-bottles.yml]: `Merge` ?

**Pull request merged**